File: mifosng-android/src/main/java/com/mifos/objects/accounts/loan/LoanApprovalRequest.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

package com.mifos.objects.accounts.loan;

/**
 * Created by ishankhanna on 23/06/14.
 */
public class LoanApprovalRequest {

    private String locale = "en";
    private String dateFormat = "dd MM yyyy";
    private String approvedOnDate;
    private String note;

    public String getLocale() {
        return locale;
    }

    public void setLocale(String locale) {
        this.locale = locale;
    }

    public String getDateFormat() {
        return dateFormat;
    }

    public void setDateFormat(String dateFormat) {
        this.dateFormat = dateFormat;
    }

    public String getApprovedOnDate() {
        return approvedOnDate;
    }

    public void setApprovedOnDate(String approvedOnDate) {
        this.approvedOnDate = approvedOnDate;
    }

    public String getNote() {
        return note;
    }

    public void setNote(String note) {
        this.note = note;
    }
}

File: core/database/src/main/java/com/mifos/core/objects/accounts/loan/LoanApprovalRequest.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.accounts.loan

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * Created by ishankhanna on 23/06/14.
 */
@Parcelize
data class LoanApprovalRequest(
    var locale: String = "en",

    var dateFormat: String = "dd MM yyyy",

    var approvedOnDate: String? = null,

    var note: String? = null,
) : Parcelable
